contracts/Vault.vy

```diff
@@ -1331,13 +1331,16 @@ def addStrategyToQueue(strategy: address):
     assert msg.sender in [self.management, self.governance]
     # Must be a current Strategy
     assert self.strategies[strategy].activation > 0
-    # Check if queue is full
-    assert self.withdrawalQueue[MAXIMUM_STRATEGIES - 1] == ZERO_ADDRESS
     # Can't already be in the queue
+    last_idx: uint256 = 0
```

Comment:
Updated strategy queue code per @fubuloubu . New code is slightly more gas efficient and is fully testable

Reply:
Should result in 1 less `SLOAD` (note currently iteration over `self.withdrawalQueue` just loads the whole thing anyways)